src/javascripts/services/transactionService.js

```diff
@@ -54,6 +54,11 @@ class TransactionService {
             return item;
         })
     }
+
+    deleteTransaction = async(id) => {
+        await this.api.deleteTransactionById(id);
+        this.transactionList = this.transactionList.filter((item) => item.id != id);
```

Comment:
Is there anyway not use filter? Because it will loop through array, that no need to do it